k8s/simple-helm/proxy/values.yaml

```diff
@@ -0,0 +1,153 @@
+# Default values for proxy.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+# This will set the replicaset count more information can be found here: https://kubernetes.io/docs/concepts/workloads/controllers/replicaset/
+replicaCount: 1
+
+# This sets the container image more information can be found here: https://kubernetes.io/docs/concepts/containers/images/
+image:
+#  repository: openremote/proxy
+  repository: 134517981306.dkr.ecr.eu-west-1.amazonaws.com/openremote/proxy
```

Comment:
You can change it back to `openremote/proxy` now that the Kubernetes changes are in the [latest release](https://github.com/openremote/proxy/releases/tag/2.9.15.0).